### PR TITLE
delete button

### PR DESCRIPTION
**Pull request by Vivek20ingh**

### CODE REVIEW

File: Components/ItemView.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const ItemView = ({item}) =>{
+  return(
+   <ListItem>
+   <Avatar source={{uri: item.avatar_url}} />
+     <ListItem.Content>
+     <ListItem.Title>{item.title}</ListItem.Title>
+     <ListItem.Subtitle>{item.year}</ListItem.Subtitle>
+   </ListItem.Content>
+   </ListItem>
```
Comment (thehellmaker) [inline Components/ItemView.js:15]: formatting is not right

File: Components/List.js [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
-            <ListItem key={i} bottomDivider>
-              
-              <Avatar source={{uri: l.avatar_url}} />
-              <ListItem.Content>
-                <ListItem.Title>{l.title}</ListItem.Title>
-                <ListItem.Subtitle>{l.year}</ListItem.Subtitle>
-              </ListItem.Content>
-            </ListItem>
-             ))}
+            list.map((l, i) => {
```
Comment (thehellmaker) [inline Components/List.js:15]: please name l and i appropriately.. Naming is very important in professsional coding. I will not understand what this is if i read this. It should be like reading an english document

File: Components/List.js [hunk 3]
```diff
@@ -1,31 +1,35 @@
...
+              if(l.ping==='0'){
```
Comment (thehellmaker) [inline Components/List.js:16]: what iss ping? Can you name it appropriately and what does'0' mean.. everthing has to be named right

**Review by thehellmaker — CHANGES_REQUESTED** [review]

File: Components/List.js (hunk 2 above)
Comment (Vivek20ingh) [reply Components/List.js:15]: okk Sir

**Pull request merged**